communication/slack-config/channels.yaml

```diff
@@ -212,6 +212,7 @@ channels:
   - name: norw-users
   - name: office-hours
   - name: opencontainers
+  - name: okteto
```

Comment:
Could you move it above by one line? otherwise lgtm

Reply:
Right. Done!